ui/cypress/e2e/explorer.test.ts

```diff
@@ -144,6 +145,34 @@ describe('DataExplorer', () => {
 
       cy.getByTestID('empty-graph--no-results').should('exist')
     })
+
+    it('can save query as task even when it has a variable', () => {
+      const taskName = 'tax'
+
+      cy.getByTestID('flux-editor').within(() => {
+        cy.get('textarea').type(
+          `from(bucket: "defbuck")
+  |> range(start: -15m, stop: now())
+  |> filter(fn: (r) => r._measurement == `,
+          {force: true}
+        )
+      })
+
+      cy.getByTestID('toolbar-tab').click()
+      cy.get('.variables-toolbar--label').click()
+
+      cy.getByTestID('flux-editor').within(() => {
+        cy.get('textarea').type(`)`, {force: true})
```

Comment:
Unclear as to why this paren is added here.  Comment?

Reply:
done, thanks!